app/controllers/flood_risk_engine/application_controller.rb

```diff
@@ -2,6 +2,8 @@ module FloodRiskEngine
   class ApplicationController < ActionController::Base
     protect_from_forgery with: :exception
 
+    layout ->(_) { 'application' }
+
```

Comment:
Yeah can do. AFAIK this engine will only ever be used for this App so going to great lengths to make it act like a generic engine not my priority